HelperMethods.java

```diff
@@ -0,0 +1,191 @@
+package datastructures;

```

Comment:
Should this be in its own package? Or still in with datastructures?

Reply:
If the CLI/HelperMethods are in the same package we can get away with not needing to import any files to them later on. If we want to abstract them we can, but it makes it much easier (from what I can see) to have it all in one.